### DISCUSSION TITLE
My site fetching is pulling data not from Transferred project

### DISCUSSION THREAD

**Discussion by jlbmagic** [supabase/supabase#29071] [Questions]
I transferred a project to a client's Supabase account. 
Since then the localhost that fetches from the project is not pulling the data in that transferred project. I can delete an entire set of records but the site will still pull those records.

The site url and the anon key are all the same.  Why is my localhost pulling from a non-existent project?

Do I need to restart a project that is transferred so that the localhost will pull from it?

**Comment by silentworks** [comment]
It's not clear what you are asking here as at one point you state that local host is not fetching from the project and at another point you say localhost is pulling from a non-existent project. Please try and give clear steps as to what is happening and what your expected outcome is.
